src/sentry/api/endpoints/project_team_details.py

```diff
@@ -1,60 +1,88 @@
-from django.http import Http404
+from drf_spectacular.utils import extend_schema
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases.project import ProjectEndpoint, ProjectPermission
+from sentry.api.exceptions import ResourceDoesNotExist
 from sentry.api.serializers import serialize
-from sentry.api.serializers.models.project import ProjectWithTeamSerializer
+from sentry.api.serializers.models.project import (
+    ProjectWithTeamResponseDict,
+    ProjectWithTeamSerializer,
+)
+from sentry.apidocs.constants import RESPONSE_FORBIDDEN, RESPONSE_NOT_FOUND
+from sentry.apidocs.examples.project_examples import ProjectExamples
+from sentry.apidocs.parameters import GlobalParams
 from sentry.models import Team
 
 
 class ProjectTeamsPermission(ProjectPermission):
     scope_map = {
-        "GET": ["project:read", "project:write", "project:admin"],
         "POST": ["project:write", "project:admin"],
-        "PUT": ["project:write", "project:admin"],
         # allow deletes with write permission because it's just removing
         # a team from a project and not anything more destructive
         "DELETE": ["project:write", "project:admin"],
     }
 
 
+@extend_schema(tags=["Projects"])
```

Comment:
I believe this should go in `Projects` and not `Teams` tab b/c the URL is a project URL and the main object we're updating is project